es6/mobify-es6.yml

```diff
@@ -188,7 +185,6 @@ rules:
   space-infix-ops: error
   space-unary-ops: error
   spaced-comment: error
-  strict: error
```

Comment:
Yeah see I don't think this is really necessary to remove? When I think of the vast majority of code we'll apply linting to (i.e. projects), we'll want to enforce that `use strict` is unnecessary.